src/functions/helpers.ts

```diff
@@ -212,7 +212,7 @@ export function toBoolean(data: FunctionResultObject | CellValue | undefined): b
         return false;
       }
     case "number":
-      return value ? true : false;
+      return !!value;
```

Comment:
it is less readable... I would stay away from !! even if it is valid javascript

Reply:
You think ? I think it was Remi that said that it's better than Boolean(value). And I think both are better than `value ? true : false`. But I do'nt care that much, I'll remove the `!!` if you do'nt like them.